cumulus-model/src/main/java/mil/army/usace/hec/cumulus/client/model/Watershed.java

```diff
@@ -0,0 +1,158 @@
+package mil.army.usace.hec.cumulus.client.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Watershed
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2021-11-02T12:49:54.974-07:00[America/Los_Angeles]")
+public class Watershed {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("office_symbol")
+    private String officeSymbol;
+
+    @JsonProperty("slug")
+    private String slug;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("area_groups")
+    private String[] areaGroups;
+
+    @JsonProperty("bbox")
+    private Integer[] bbox;
+
+    /**
+     * Get Id of Watershed
+     *
+     * @return
+     */
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * Get office symbol for Watershed
+     *
+     * @return
+     */
+    public String getOfficeSymbol() {
+        return officeSymbol;
+    }
+
+    public void setOfficeSymbol(String officeSymbol) {
+        this.officeSymbol = officeSymbol;
+    }
+
+    /**
+     * Get Watershed slug
+     *
+     * @return
+     */
+    public String getSlug() {
+        return slug;
+    }
+
+    public void setSlug(String slug) {
+        this.slug = slug;
+    }
+
+    /**
+     * Get Watershed name
+     *
+     * @return
+     */
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Get Watershed's area groups
+     *
+     * @return
+     */
+    public String[] getAreaGroups() {
+        return areaGroups;
+    }
+
+    public void setAreaGroups(String[] areaGroups) {
+        this.areaGroups = areaGroups;
+    }
+
+    /**
+     * Get Watershed bbox
+     *
+     * @return
+     */
+    public Integer[] getBbox() {
+        return bbox;
+    }
+
+    public void setBbox(Integer[] bbox) {
+        this.bbox = bbox;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
not sure if we want to stick with this for equality or just check the id

Reply:
If watershed Ids are unique then I think we just check Id. Maybe also check offices are the same?

Reply:
Updated equality to just check against the Ids